insights/parsers/pcs_config.py

```diff
@@ -136,3 +138,15 @@ def parse_content(self, content):
 
     def get(self, key):
         return self.data.get(key)
+
+    @property
+    def cluster_props_dict(self):
+        """
+        (dict): This function returns a dict of all cluster property and value
+        """
+        prop_result = {}
+        if "Cluster Properties" in self.data and self.data.get("Cluster Properties"):
+            for property_line in self.data.get("Cluster Properties"):
+                key, _, value = property_line.partition(":")
+                prop_result[key.strip()] = value.strip()
+        return prop_result
```

Comment:
@chenlizhong would it make sense to compute this as an attribute, once, in `parse_content`?

Reply:
@chenlizhong - better to make it as an attribute, or move the code into the `parse_content` to avoid the computation in every calling of it.

Reply:
@chenlizhong - if you forget to `git add` your updates to the last commit? Seems nothing is changing in that commit.